x-pack/plugin/esql/qa/testFixtures/src/main/resources/lookup-join.csv-spec

```diff
@@ -325,3 +324,80 @@ count:long | type:keyword
 3          | Success
 1          | Disconnected
 ;
+
+nonUniqueLeftKeyOnTheDataNode
```

Comment:
I've been grouping all tests that work with employees-join-languages at the top of the file, and joins against clientips and message-types following in blocks of tests. Perhaps we can move this to before the clientips tests, just after the languages tests, so it aligns with my PR (and therefore reduces conflicts too).